scripts/onnx/test.sh

```diff
@@ -66,23 +68,26 @@ if [[ "${SHARD_NUMBER}" == "1" ]]; then
     --ignore "$top_dir/test/onnx/test_pytorch_onnx_caffe2_quantized.py" \
     "${test_paths[@]}"
 
+  # Tests that cannot run in parallel.
   pytest "${args[@]}" \
+    "$top_dir/test/onnx/test_pytorch_onnx_caffe2.py" \
+    "$top_dir/test/onnx/test_pytorch_onnx_caffe2_quantized.py"
+
+  pytest "${args[@]}" "${args_parallel[@]}" \
     "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset7" \
     "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset8" \
     "$top_dir/test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime_opset9" \
     "$top_dir/test/onnx/test_custom_ops.py" \
     "$top_dir/test/onnx/test_models_onnxruntime.py" \
     "$top_dir/test/onnx/test_utility_funs.py" \
-    "$top_dir/test/onnx/test_pytorch_onnx_caffe2.py" \
-    "$top_dir/test/onnx/test_pytorch_onnx_caffe2_quantized.py" \
     "$top_dir/test/onnx/test_pytorch_onnx_shape_inference.py" \
     "$top_dir/test/onnx/test_onnx_export.py"
```

Comment:
Is it possible to let pytest discover all tests under `$top_dir/test/onnx/`, minus the two caffe2 files? This way it would be much easier moving tests around and creating new tests

Reply:
I see we are using two shards. That said, would it be possible to let one machine work on some specified tests, and the other the rest of the tests, whatever those tests are?

Reply:
e.g. `--ignore="$top_dir/test/onnx/test_pytorch_onnx_caffe2.py" --ignore="$top_dir/test/onnx/test_pytorch_onnx_caffe2_quantized.py" --ignore="$top_dir/test/onnx/test_pytorch_onnx_caffe2_quantized.py"`

Reply:
Sorry I misread the first command